recipes/flowtorch/conda-forge.yml

```diff
@@ -0,0 +1,17 @@
+channel_priority: strict
```

Comment:
This needs to be removed again as mentioned in my other comment.

Reply:
Same as [this comment](https://github.com/conda-forge/staged-recipes/pull/17175#discussion_r764818289).